src/runner/nunit.runner/Properties/AssemblyCommon.cs

```diff
@@ -26,12 +26,12 @@
 
 [assembly: AssemblyCompany("NUnit Software")]
 [assembly: AssemblyProduct("NUnit 3 Xamarin Runner")]
-[assembly: AssemblyCopyright("Copyright (C) 2015 Charlie Poole")]
+[assembly: AssemblyCopyright("Copyright (C) 2017 Charlie Poole")]
 [assembly: AssemblyTrademark("NUnit is a trademark of NUnit Software")]
 [assembly: AssemblyCulture("")]
 
-[assembly: AssemblyVersion("3.0.*")]
-[assembly: AssemblyFileVersion("3.0.0.0")]
+[assembly: AssemblyVersion("3.6.*")]
```

Comment:
Seems sensible to me. I wasn't sure why they were *'s - so I just left them as they were!